### PR TITLE
Issue #2878 - Fixing aggregated javadoc jar

### PR DESCRIPTION
**Pull request by olamy**
The idea behind this change is to build/deploy the aggregated javadoc as part of the release

### CODE REVIEW

**Comment by olamy** [comment]
buid is currently failing because of https://issues.apache.org/jira/browse/MNG-6590 
fixed with maven 3.6.1

File: pom.xml [hunk 1]
```diff
@@ -1297,23 +1295,28 @@
...
+      <id>aggregate-javadoc-jar</id>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-javadoc-plugin</artifactId>
+              <configuration>
+                <attach>true</attach>
+                <excludePackageNames>com.acme,*.jmh*,org.eclipse.jetty.embedded*,org.eclipse.jetty.example.asyncrest*,org.eclipse.jetty.test*</excludePackageNames>
```
Comment (joakime) [inline pom.xml]: Why is this a profile?
Can't this just be a `<pluginManagement>` or `<plugin>` configuration instead?

File: jetty-jmh/src/main/java/org/eclipse/jetty/http/jmh/MultiPartBenchmark.java [hunk 2]
```diff
@@ -277,6 +295,191 @@ public static void main(String[] args) throws RunnerException
         
         new Runner(opt).run();
     }
+
+
+    public static class MultipartExpectations
```
Comment (joakime) [inline jetty-jmh/src/main/java/org/eclipse/jetty/http/jmh/MultiPartBenchmark.java]: This seems unrelated to the PR.

Comment (joakime) [reply jetty-jmh/src/main/java/org/eclipse/jetty/http/jmh/MultiPartBenchmark.java]: Ah, i get it now.
You moved this from jetty-http to jetty-jmh to break the dependency that javadoc freaks out about.

File: pom.xml (hunk 1 above)
Comment (joakime) [reply pom.xml]: I went ahead and just made these configs standard
